docs/source/caching.rst

```diff
@@ -0,0 +1,270 @@
+=================
+Caching
+=================
+
+Resolve Caching
+===============
+
+Resolve caching is a feature that caches resolves to a memcached (in-memory) server. Because the server is in-memory,
+the full contents of the cache are lost if the memcached service shuts down by any means.
+
+Cache contents
+--------------
+The following information is stored to the memcached server for each solve:
+
+* Solver information about the previously cached solve.
+* Release times information about when each package variant in the resolve was last released.
+* Variant states information about the state of a variant. For example, in the 'filesystem' repository type, the 'state' is the last modified date of the file associated with the variant (perhaps a package.py). If the state of any variant has changed from a cached resolve - eg. if a file has been modified - the cached resolve is discarded.
+
+Setup
+-----
+
+To enable memcached caching, you need to configure the :data:`memcached_uri` config variable. This variable accepts a list of memcached uri servers or None. Example with memcached running on localhost on its default port:
+
+.. code-block:: console
+
+   memcached_uri = ["127.0.0.1:11211"]
+
+This is the only parameter you need to configure to enable caching of the content and location of package file definitions and resolutions in Rez.
+
+Configuration
+-------------
+There are several variables that can be accessed to modify the default behavior of resolve caching:
+
+* :data:`resolve_caching`
+* :data:`cache_package_files`
+* :data:`cache_listdir`
+* :data:`resource_caching_maxsize`
+* :data:`memcached_package_file_min_compress_len`
+* :data:`memcached_context_file_min_compress_len`
+* :data:`memcached_listdir_min_compress_len`
+* :data:`memcached_resolve_min_compress_len`
```

Comment:
```suggestion
Please refer to the :ref:`caching` configuration section for a complete list of settings.
```
?

Reply:
@JeanChristopheMorinPerso To make sure I'm understanding this correctly, the `:ref` is just a cross reference directive right? https://www.sphinx-doc.org/en/master/usage/referencing.html#role-ref

If so, this doesn't seem to be working for some reason. Maybe because the new page's heading is also "Caching"? I'm not sure.